Dockerfile

```diff
@@ -1,32 +1,38 @@
-FROM       python:3
+FROM python:3-alpine
```

Comment:
Read this https://pythonspeed.com/articles/alpine-docker-python/

Reply:
I have heard such claims but have never been able to reproduce them with a real project, my time building the alpine based image is 4 minutes and building a debian based one is 2 minutes but the latter already has gcc and all other deps installed so ofc it is a touch faster but the size of my alpine image was ~250mb wheras with just plain `python:3` image it was 1.16Gb but due to the runtime concerns I cannot outright disprove I have reverted back to that image as the base.